.dockerignore

```diff
@@ -0,0 +1,3 @@
+.env
+./env.template
+.venv
```

Comment:
You could include more here, but you have the most important things, and you actually copy specific files/folders so much stuff will be copied in incorrectly.

The only thing you are currently copying that could be ignored are the `__pycache__` folders

Reply:
done